spec/openapi.yml

```diff
@@ -167,6 +167,42 @@ components:
           items:
             $ref: "#/components/schemas/SandboxLog"
 
+    SandboxMetric:
+      description: Metric entry with timestamp and line
+      required:
+        - timestamp
+        - cpuTotal
```

Comment:
It's in required, but it isn't in properties, also it isn't implemented in API server